lib/debezium/schema.go

```diff
@@ -155,6 +157,10 @@ func (f Field) ToKindDetails() typing.KindDetails {
 	case Array:
 		return typing.Array
 	default:
+		if f.Type != "" || f.DebeziumType != "" {
```

Comment:
Are `f.Type` or `f.DebeziumType` ever empty strings?

Reply:
We did have a case of it being empty here: https://github.com/artie-labs/reader/pull/473

Reply:
Just thinking more about this, my eventual goal is to have this return an error instead of `typing.Invalid`.

Doing conditional checks on empty string will still require me to check who is passing empty strings. I just removed it so I can track down any potential codepaths that are emitting empty or unhandled types.

Reply:
Would it be useful to also log if `f.DebeziumType` is not empty after the `switch f.DebeziumType` statement has completed? In other words, if `f.DebeziumType` is present do we always expect that the case statement should handle it? Should it be an error if `f.DebeziumType` = `"junk"` and `f.Type` = "string"?

Reply:
I'm down

Reply:
Let's log it.

> Should it be an error if f.DebeziumType = "junk" and f.Type = "string"?

I think so